### ISSUE TITLE
Can't return from inside while loop

### ISSUE THREAD

**Issue by healeycodes** [ghost-language/ghost#92]
Hi!

This language looks cool! I ran into a bug (maybe?) while trying it out.

I wasn't able to return from inside a while loop – I read https://ghostlang.org/docs/nightly/control-flow but it wasn't clear if this should be allowed or if this is expected behaviour.

The following example enters an infinite loop on https://play.ghostlang.org/

```ghost
function returnFromWhile() {
    while (true) {
        // We never return here
        return "returned from inside while"
    }
}

returnFromWhile()
```

**Comment by kaidesu** [comment]
Thanks for reporting! I'll get this resolved and some tests written~

(btw, huge fan of your blog)

**Comment by kaidesu** [comment]
Fixed in 8947686555562117832bc023f9287705181a4685

I may end up changing this soon for a `break` statement instead, but for now this'll do. Just now at the point where I'm working with the language directly, seeing how it feels to write/run programs.

Note that I've rebuilt the interpreter from scratch in the `1.0` branch where the fix is located - the version on the playground is still on an older build for the time being as I work on wrapping up tests.

You can find a quick ref docs page here: https://github.com/ghost-language/ghost/blob/1.0/DOCS.md
And a list of various example programs in the `examples` directory: https://github.com/ghost-language/ghost/tree/1.0/examples

Thanks for giving Ghost a spin!
